src/pages/en/guides/cms/contentful.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Contentful & Astro
+description: How to connect your Contentful CMS to your Astro site.
+layout: ~/layouts/MainLayout.astro
+---
+
+[Contentful](https://www.contentful.com/) is a headless CMS that allows you to manage content, integrate with other services, and publish to multiple platforms. It is a great choice for Astro projects that need to pull in content from a CMS.
+
+## Prerequisites
+
+- **An Astro project** - If you don't have an Astro project yet, our [Installation guide](/en/install/auto/) will get you up and running in no time.
```

Comment:
Maybe something like "for this walkthrough, you can choose `Empty Project` as the starter template."